### PR TITLE
Connytestar

### PR DESCRIPTION
**Pull request by connybackas**
Started implementation for velocity calculation

### CODE REVIEW

**Comment by connybackas** [comment]
New Json format:
 {
     "gpsPoint":{"timeStamp":"15:02:13", "latitud": 13, "longitud": 37},
    "direction": 90
 }

File: src/main/java/fi/robosailboat/webservice/calculations/VelocityCalculator.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    public VelocityCalculator(GpsPoint first, GpsPoint second) {
+        first = first;
+        second = second;
+    }
+
+    public String calculateKmh(){
+        //Calculate velocity and return kmh
+        return "";
+    }
+
```
Comment (MrNorthLake) [inline src/main/java/fi/robosailboat/webservice/calculations/VelocityCalculator.java]: typo "calculate"

File: src/main/java/fi/robosailboat/webservice/calculations/VelocityCalculator.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package fi.robosailboat.webservice.calculations;
```
Comment (MrNorthLake) [inline src/main/java/fi/robosailboat/webservice/calculations/VelocityCalculator.java]: Maybe some documentation before the class, about what this is and so on.. so the next programmer knows directly that this class does

File: src/main/java/fi/robosailboat/webservice/boatCommunication/GpsPoint.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package fi.robosailboat.webservice.boatCommunication;
+
```
Comment (MrNorthLake) [inline src/main/java/fi/robosailboat/webservice/boatCommunication/GpsPoint.java]: is this class neccessary?

**Pull request closed without merging**